### DISCUSSION TITLE
Alarm sound repeats continuosly

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#3731] [Q&A]
Hello.
I have just switched to a new phone and I'm experiencing the issue described in [this](https://github.com/NightscoutFoundation/xDrip/issues/1418) thead, which has already been closed.
Basically, I come from a Redmi Note 10 phone and now I'm using a Motorola g84; I replicated the xDrip settings using the qr-code.
Now, every time a glucose alarm is raised the sound is just repeated continuosly until it is snoozed.
This happens even using the "test alert" button, but it doesn't happen with other alarms such as the "Bg rising fast".
![Screenshot_20241024-105901](https://github.com/user-attachments/assets/811010a6-ef7e-4d8e-87b4-3b6a3bd9bae6)

My phone is updated to Android 14 and I'm using the last xDrip nightly.
In the original thread someone suggested that the bug might be related to Samsung phones, but obviously it's not my case.
I'm going to share all details if needed.

**Comment by Navid200** [comment]
Can you post a screenshot of the classic status page?

**Reply by dagord** [reply]
Sure!
![Screenshot_20241024-134810](https://github.com/user-attachments/assets/3b9a68ec-56c6-4bda-b12c-dd2d4e7076f0)

**Reply by Navid200** [reply]
Can you go to the logs? Tap on `Save Logs` at the bottom.
Then, go to Download folder on your phone.  You will see a folder called "xDrip-export".
Inside, you will find a file called "xDrip-log.txt".
View it and post a screenshot of the first 4 lines.

**Reply by dagord** [reply]
Not sure if this is what you are looking for
![Screenshot_20241024-141113](https://github.com/user-attachments/assets/22929f47-d398-4a79-a0d6-ce1aa34d5876)

**Reply by Navid200** [reply]
Can you let us hear it loop?  Can you record the xDrip alert and share?

**Reply by dagord** [reply]
Ok, I'm trying to attach the recording...

https://github.com/user-attachments/assets/4ca96b1b-2d65-4498-81af-209d4f869805

As you can hear, the sound is played continuously, but if I choose the same sound for, e.g., "bg rising fast" it's reproduced just once.

**Reply by Navid200** [reply]
What happens if you choose the default xDrip sound for this alert?

**Reply by dagord** [reply]
Bingo! The default sound works.
So I decided to choose a "custom sound" and it works as well.
The issue seems to affect only the "system sounds", so maybe it's an issue probably related to Android instead of xDrip.

**Reply by Navid200** [reply]
I'm happy to consider this addressed if you so desire.

But, since this has been coming up and it is possible to come up again, my next question was going to be if you were willing to share the sound file that caused the problem.  Or, if you are able, you can inspect it yourself.
I wonder if there is a setting in the file that instructs some media players to loop.  

It's up to you.

**Reply by dagord** [reply]
It's not a problem with a single file: the issue affects all the "system sounds". I can't locate the exact directory where these files live in, but I suspect that if I could copy them to another location and use them as "custom sounds" within xDrip, they would work as expected
